CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+project(TOY_MT_FRAMEWORK)
+cmake_minimum_required(VERSION 2.8)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules")
+find_package(OpenMP)
+find_package(TBB)
+find_package(CppUnit)
+find_package(Boost)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
+
+add_subdirectory(BusyWaitCalibration)
+add_subdirectory(DispatchProcessingDemo)
```

Comment:
Here’s the back-and-forth we need. I don’t know what dispatch is, so I didn’t know how to look for it. What is it?

—Jim

------------------------
James Amundson
Head, Scientific Software Infrastructure Department
Scientific Computing Division, Fermilab
+1 (630) 840-2430



On Mar 28, 2016, at 10:12 AM, Chris Jones <notifications@github.com<mailto:notifications@github.com>> wrote:


In CMakeLists.txt<https://github.com/Dr15Jones/toy-mt-framework/pull/1#discussion_r57582155>:

> @@ -0,0 +1,20 @@
> +project(TOY_MT_FRAMEWORK)
> +cmake_minimum_required(VERSION 2.8)
> +
> +set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules")
> +find_package(OpenMP)
> +find_package(TBB)
> +find_package(CppUnit)
> +find_package(Boost)
> +
> +set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
> +
> +add_subdirectory(BusyWaitCalibration)
> +add_subdirectory(DispatchProcessingDemo)


I assume something like if (DISPATCH_FOUND) is also needed?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/Dr15Jones/toy-mt-framework/pull/1/files/63c137130425ce1478f6dd9dd75fb351723cefc5#r57582155>